server/server.go

```diff
@@ -0,0 +1,213 @@
+package server
+
+import (
+	"errors"
+	"expvar"
+	"net"
+	"net/http"
+	"net/http/pprof"
+	"os"
+	"path/filepath"
+
+	"github.com/boltdb/bolt"
+	containers "github.com/containerd/containerd/api/services/containers/v1"
```

Comment:
this is already the package name, isn't it?

If renaming it, I'd rather have it called `containersapi` as it's been done elsewhere in the repo so far.